my-app/src/components/Body/Taskzone/Zone/Zone.jsx

```diff
@@ -1,18 +1,18 @@
-import "./Task.scss";
+import Card from "./Card/Card";
+import "./Zone.scss";
 
-let Task = (props) => {
+let Zone = (props) => {
```

Comment:
Hint: you can destructure props so that you don't need to have `props.` prefix for all variables in the following lines.

For instance:
```
let Zone = ({ id, zoneName, cards }) => {
// ...
  <div>{zoneName}</div>
// ...
  {cards.map((card) => {
// ...
```